env.example

```diff
@@ -21,21 +21,21 @@ REDIS_PORT=6379
 # MOCK_REDIS=1 will use an in-memory, mock Redis instance. Useful for testing.
 MOCK_REDIS=
 
-#https://developers.onelogin.com/saml - is used for this current portion of code
+#https://medium.com/disney-streaming/setup-a-single-sign-on-saml-test-environment-with-docker-and-nodejs-c53fc1a984c9 - is used for this current portion of code
 
-#OICD_BASE_URI = BASE URL e.g.  https://openid-connect.onelogin.com/oidc
-OICD_BASE_URI=""
+#SAML2 = BASE URL e.g.  http://localhost/login/callback
+SAML2_BASE_URI=""
 
-#OIDC_CLIENT_ID = CLIENT ID obtained from OpenId Connect Strategy - THIS WILL BE OBTAINED FROM SENECA AT A LATER DATE.
-OIDC_CLIENT_ID=
+#SAML2_CLIENT_ID = CLIENT ID obtained from SAML Strategy default: saml-poc - THIS WILL BE OBTAINED FROM SENECA AT A LATER DATE.
```

Comment:
You can get rid of all the `THIS WILL BE OBTAINED FROM SENECA AT A LATER DATE.` notes.